tools/eslint-config/package.json

```diff
@@ -26,21 +26,21 @@
     "rules/"
   ],
   "engines": {
-    "node": ">= 8"
+    "node": ">=10"
   },
   "peerDependencies": {
     "eslint": ">=6.8.0",
-    "prettier": ">=1.19.1"
+    "prettier": "^1.19.1 || ^2.0.2"
```

Comment:
I'd keep `^2.0.2`, otherwise the `engine` has less relevance

Reply:
I prefer to remove v1 because some inconsistences with v2, I kept it for compatibility until migration, but let's do this :stuck_out_tongue_winking_eye: